OpenShift/helm/nexus-iq/Chart.yaml

```diff
@@ -1,9 +1,25 @@
-apiVersion: v1
-appVersion: 1.84.0
-description: A Helm chart for Kubernetes
-name: iqserver
-version: 0.1.0
+apiVersion: v2
+name: nexus-iq-server
```

Comment:
when you change the name from `iqserver` to `nexus-iq-server`, won't you have to change all of the variables in all of the templates? I see they're still using things like `iqserver.fullname`. I'm a little confused at how this is working now.